scripts/Dockerfile.verify

```
@@ -0,0 +1,65 @@
+#   local — install from packed tarballs (pre-publish verification)
+#   npm   — install from npm registry at runtime (post-publish verification)
+#
+# Build:
+#   docker build -f scripts/Dockerfile.verify --target local -t rundown-verify:local .
+#   docker build -f scripts/Dockerfile.verify --target npm   -t rundown-verify:npm .
+
+# ── Base stage ───────────────────────────────────────────────────────────────
+
+FROM node:22-slim AS base
+
+RUN apt-get update && \
```

Hadolint (2.14.0)

[warning] Pin versions in apt get install. Instead of `apt-get install <package>` use `apt-get install <package>=<version>`

(DL3008)